Utils/Dataflow/025_chicagoES/stage.py

```diff
@@ -353,12 +362,16 @@ def main(args):
     stage.set_input_message_type(messageType.JSON)
     stage.set_output_message_type(messageType.JSON)
 
+    stage.set_default_arguments(config=os.path.join(base_dir, os.pardir,
+                                                    'config', '025.cfg'))
+
     stage.process = process
 
     exit_code = 0
     exc_info = None
     try:
         stage.configure(args)
+        init_es_client(stage.CONFIG['ChicagoES'])
```

Comment:
Commit message states that this commit merely "allows" something - however, this line means that the ES client is now initialized before starting stage, rather than when `get_es_client()` is called for the first time. This explicit initialization has nothing to do with "allowing" and changes in `init_es_client()`, right?

Reply:
@Evildoor, thank you for pointing to it: this part of the changes was not only unexplained, but also lead to some problems with the stage `--skip` mode.

I have fixed those issues (0f70241...bc8645d) and reworded the initial commit (0f70241) -- please take a look.

Reply:
This particular issue - `init_es_client()` not mentioned in commit logs - was fixed. Will comment on other commits and close this conversation as resolved some time later.